hyrisecockpit/database_manager/worker_pool.py

```diff
@@ -1,13 +1,19 @@
 """The WorkerPool object represents the workers."""
-from multiprocessing import Event, Process, Queue, Value
 from multiprocessing.synchronize import Event as EventType
+from sys import platform
 from typing import List, Optional
 
 from apscheduler.schedulers.background import BackgroundScheduler
 
 from .cursor import ConnectionFactory
 from .worker import execute_queries, fill_queue
 
+if platform.startswith("linux"):
+    from multiprocessing import Event, Process, Queue, Value
+elif platform.startswith("darwin"):
+    from multiprocessing import Event, Process, Value
+    from hyrisecockpit.cross_platform_support.mac_osx_support import MacQueue as Queue  # type: ignore
```

Comment:
How about we import platform-specific components like this:
```suggestion
from multiprocessing import Event, Process, Value
from hyrisecockpit.cross_platform_support.mac_osx_support import Queue
```
and handle the if/else in the `mac_osx_support.py` file?
This way, we would only declare platform specifics once in a single file.

Reply:
I like the idea. How about to rename `mac_osx_support ` to `cross_platform_support` and import all cross-platform critical modules from there?